PostHog Array 1.30.0 (#2433)
PostHog/posthog.com: contents/blog/the-posthog-array-1-30-0.md

---
date: 2021-11-17
title: Array 1.30.0
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["Release notes"]
featuredImage: ../images/blog/array/1-28-0.png
excerpt: Excerpt here
---

> PostHog 1.30.0 includes a description here.

Reviewer: @paolodamico Who needs to be filling this in?

Author: Done!

## PostHog 1.30.0 release notes

> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).

**Release highlights:**
- [Fresh new look-and-feel](#fresh-new-look-and-feel)
- [Correlation analysis](#correlation-analysis)
- [Saved insights](#saved-insights)
- [Fully revamped recordings](#fully-revamped-recordings)

### Fresh new look-and-feel

> PostHog just got a lot fresher! We have a brand new UI and layout that had been on the works, codename `lemonade` (because it's fresh). It's not only a coat of paint, we've pushed a lot of improvements to overall navigation and performance, we call it turbo mode.

Reviewer: ```suggestion
PostHog just got a lot fresher! We have a brand new UI and layout that had been in the works, codenamed `lemonade` (because it's fresh). It's not only a new coat of paint - we've also pushed a lot of improvements to overall navigation and performance too. We call it turbo mode.
```

> {images here}

Author: @clarkus I know you've put together some images recently (I believe for the launch), can you share the images we should add for this array pls?

Author: @Twixes if you want to review this (or the entire thing too 😉)

Reviewer 2: How's this @paolodamico? I have many more options at https://www.figma.com/file/NHPA2hVWwaKp4eTGWdigs9/?node-id=0%3A1 

![schenanigans header tan](https://user-images.githubusercontent.com/254612/141598518-fe832b04-4e95-4899-9911-7c0a0a3e60a3.png)

Reviewer 3: I would strongly prefer screenshots over mockups here. I can take some showing off the UI refresh.

Author: That makes sense to me @Twixes! I'll ping you on Slack to get some cool screenshots

Reviewer 3: Alright, here's a composition showing off a few aspects of the new UI:
<img width="1527" alt="lemonade" src="https://user-images.githubusercontent.com/4550621/142001323-aa0f4d8c-229f-488a-9e7e-89a24c8c5762.png">

Reviewer 3: Oops, the mobile screenshot was a bit wrong, now correct:
<img width="1527" alt="Lemon Composition" src="https://user-images.githubusercontent.com/4550621/142002267-7f91887f-4415-4445-bbad-6dfeb65be993.png">

<br />

### Correlation analysis

> Trying to understand why your users are converting or dropping? Presenting Correlation analysis. This nifty new insight will automatically match your funnels to any relevant signals on why your users convert or drop (e.g. "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert"). Funnel optimization to the next level. [Read more here](/docs/....).

Author: @neilkakkar would be awesome if we had some docs for the release on Wed

Reviewer 4: Neil is out this week. @hazzadous can help perhaps?

Reviewer 5: Yes can add something. Do you have logs that you think our great that I can use as a template @paolodamico ?

Reviewer 5: @paolodamico will something like this do? https://github.com/PostHog/posthog.com/pull/2445/files

Author: This is great @hazzadous! Added some comments and will link to the new place in the docs already

> Trying to understand why your users are converting or dropping? Presenting Correlation analysis. This nifty new insight will automatically match your funnels to any relevant signals on why your users convert or drop (e.g. "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert"). Funnel optimization to the next level. [Read more here](/docs/....).

Reviewer: ```suggestion
Want to understand why users convert or churn? Presenting: Correlation analysis. This nifty new insight automatically matches funnels to any relevant conversion signals, giving you effortless correlation information such as "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert". This is a very powerful which enables you to take funnel optimization to the next level. 

[Read more here](/docs/....).
```

{image here}

> 🎁 Correlation analysis is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).

<br />

### Saved insights

> No more recreating the same insight multiple times. You will now be able to save insights on PostHog without adding them to a dashboard. Further, you'll be able to see a list of insights created by other team members, and easily search them and filter them. We're investing a lot of effort to make sure you can better collaborate within PostHog.

Reviewer: ```suggestion
Tired of creating the same insights multiple times? You can now save insights on PostHog without adding them to a dashboard. Further, you're able to see, search and filter a list of insights created by other team members - which makes it a lot easier to collaborate with PostHog. 
```

{image here}

<br />

### Fully revamped recordings

> The recordings playback experience just got a lot better. From significant performance improvements (you'll no longer need to wait for the entire recording to load), to a brand new playback interface. Find the right spot in a recording quickly and understand better what your users are doing.

Author: @alexkim205 @rcmarron can you help me review this?

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-recordings.png" alt="Screenshot of new recordings playback experience" />

<br />

> 

Author: @macobo do you want to include something about groups? maybe to allow users to start capturing group info?

Reviewer: Decision is up to you both, but @macobo and I are chatting later this week about a launch for Groups. My expectation at the moment is that I'll work on a separate blog post for it, probably writing next week.

Reviewer 4: We'll handle it later. This release we'll be testing with alpha users.

### Other improvements & fixes
- **Turbo mode**. You'll notice a significant speed improvement when using PostHog. App navigation will now happen almost instantly. In particular, you'll notice navigation between dashboards and insights happens without any delay.

> - **Duplicate dashboards**. Thanks to a community member you can now easily duplicate your dashboard instead of manually recreating it.

Reviewer: It would be good to credit the community member here?

```suggestion
- **Duplicate dashboards**. Thanks to community member [Yuvaraj J](https://github.com/PostHog/posthog/pull/6476), you can now easily duplicate dashboards instead of manually recreating them. Thanks, Yuvaraj!
```

- Fixed bugs with person counts not matching between an insight graph and the person list.
- Significantly faster frontend builds as we transitioned from `webpack` to `esbuild`.
- Fixed a bug that prevented creating cohorts from trends.
- Success and error toast alerts will now show at the bottom of the screen so it no longer covers critical elements in the page.
- Plus **350+ more** improvements & fixes.

### Deprecation & removal notices

1. We're now fully removing the legacy Sessions list page. Read more about it, [in this blog post](/blog/sessions-removal).

### Help us improve PostHog

We’re working hard to improve the PostHog experience and would love to talk to you! Please join one of our Product, Engineering, or Marketing team members on a quick 30-min call to help us understand how to improve. Schedule directly [on Calendly](https://calendly.com/posthog-feedback).

As a small thank you for your time, we're giving away awesome [PostHog merch](https://merch.posthog.com)!

## PostHog News

No new joiners this month, but stay tuned for new people joining real soon!

## Community
### Community MVP 🏆

> Thanks to all our community members for helping move PostHog forward! This release cycle's Community MVP goes to [timgl](https://github.com/asherf)!

Reviewer: ```suggestion
Thanks to all our community members for helping move PostHog forward! This release cycle's Community MVP goes to [asherf](https://github.com/asherf)!
```

Pretty sure Tim isn't who we should celebrate here lol

---
date: 2021-11-17
title: Array 1.30.0
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["Release notes"]
featuredImage: ../images/blog/array/1-30-0.png
excerpt: Brand new and faster user interface, automatic conversion signal detection with Correlation analysis, saving insights for future use and a fully revamped recordings playback experience and more than 350 improvements/fixes more.
---

PostHog 1.30.0 is a milestone release! A brand new and faster user interface, automatic conversion signal detection with Correlation analysis, saving insights for future use and a fully revamped recordings playback experience are just a few of the improvements included.

## PostHog 1.30.0 release notes

> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).

**Release highlights:**
- [Fresh new look-and-feel](#fresh-new-look-and-feel)
- [Correlation analysis](#correlation-analysis)
- [Saved insights](#saved-insights)
- [Fully revamped recordings](#fully-revamped-recordings)

### Fresh new look-and-feel

> PostHog just got a lot fresher! We have a brand new UI and layout that had been in the works, codenamed `lemonade` (because it's fresh). It's not only a new coat of paint - we've also pushed a lot of improvements to overall navigation and performance too. We call it turbo mode.
> 
> 
> <img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-lemonade.png" alt="Screenshot of new user interface" />

Reviewer 3: Copy suggestion:
```suggestion
We reworked PostHog's UI philosophy from the ground up, all to offer you the most intuitive and sleek user experience possible. With 1.30.0 we're introducing our redesigned navigation experience – codenamed Lemonade for its freshness. An evolution of the interface you know, it's simply much nicer packaging for existing features, along with a new helpful addition: breadcrumbs for hierarchical navigation. We hope you'll find this a joy to use. And if you have any feedback regarding the redesign, we'd love to hear your thoughts!

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-lemonade.png" alt="Screenshot of new user interface" />

In addition to a new coat of paint, we've been working on performance improvements – codenamed Turbo Mode. Although not visible at first glance, recent under-the-hood changes make switching between pages feel smoother and snappier.
```

---
date: 2021-11-17
title: Array 1.30.0
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["Release notes"]
featuredImage: ../images/blog/array/1-30-0.png
excerpt: Brand new and faster user interface, automatic conversion signal detection with Correlation analysis, saving insights for future use and a fully revamped recordings playback experience and more than 350 improvements/fixes more.
---

> PostHog 1.30.0 is a milestone release! A brand new and faster user interface, automatic conversion signal detection with Correlation analysis, saving insights for future use and a fully revamped recordings playback experience are just a few of the improvements included.

Reviewer: ```suggestion
PostHog 1.30.0 is a milestone release! We've introduced a brand new, faster user interface, automatic conversion signal detection with correlation analysis, the ability to save insights for future use and a fully revamped recordings playback experience. And that's just for starters!
```

## PostHog 1.30.0 release notes

> > For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).

Reviewer: ```suggestion
> Don't see the new features on your self-hosted deployment? Remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
```

- [Fresh new look-and-feel](#fresh-new-look-and-feel)
- [Correlation analysis](#correlation-analysis)
- [Saved insights](#saved-insights)
- [Fully revamped recordings](#fully-revamped-recordings)

### Fresh new look-and-feel

We reworked PostHog's UI philosophy from the ground up, all to offer you the most intuitive and sleek user experience possible. With 1.30.0 we're introducing our redesigned navigation experience – codenamed Lemonade for its freshness. An evolution of the interface you know, it's simply much nicer packaging for the existing features, along with a new helpful addition: breadcrumbs for hierarchical navigation. We hope you'll find this a joy to use. And if you have any feedback regarding the redesign, we'd love to hear your thoughts!

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-lemonade.png" alt="Screenshot of new user interface" />

In addition to the new coat of paint, we've been working on performance improvements – codenamed Turbo Mode. Although not visible at first glance, recent under-the-hood changes make switching between pages feel smoother and snappier.

<br />

### Correlation analysis

> Want to understand why users convert or churn? Presenting: Correlation analysis. This nifty new insight automatically matches funnels to any relevant conversion signals, giving you effortless correlation information such as "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert". This is a very powerful which enables you to take funnel optimization to the next level. 

Reviewer: ```suggestion
Want to understand why users convert or churn? Presenting: Correlation analysis. This nifty new insight automatically matches funnels to any relevant conversion signals, giving you effortless correlation information such as "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert". This is a very powerful feature which enables you to take funnel optimization to the next level. 
```

Read more on the [Correlation analysis docs](/docs/user-guides/correlation).

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-correlation.png" alt="Screenshot of correlation analysis" />

> 🎁 Correlation analysis is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).

<br />

### Saved insights

Tired of creating the same insights multiple times? You can now save insights on PostHog without adding them to a dashboard. Further, you're able to see, search and filter a list of insights created by other team members - which makes it a lot easier to collaborate with PostHog. 

<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-saved-insights.png" alt="Screenshot of saved insights" />

<br />

### Fully revamped recordings

> The recordings experience just got a lot better. We added a new recordings tab that allows you to filter and search across multiple days (replacing the [old sessions tab](/blog/sessions-removal)). Once you've found the recording you want to watch, there's a brand new player experience that loads much faster and overlays events on the seekbar. Find the right spot in a recording quickly and understand better what your users are doing.

Reviewer: ```suggestion
The recordings experience just got a lot better. We added a new recordings tab that enables you to filter and search across multiple days (replacing the [old sessions tab](/blog/sessions-removal)). Once you've found the recording you want to watch, there's a brand new player experience that loads much faster and overlays events on the seekbar. Find the right spot in a recording quickly and understand better what your users are doing.
```
